qiita_pet/templates/list_studies.html

```diff
@@ -6,7 +6,6 @@
 
 <script src="{% raw qiita_config.portal_dir %}/static/vendor/js/select2.min.js"></script>
 <script src="{% raw qiita_config.portal_dir %}/static/js/sharing.js"></script>
-<script src="{% raw qiita_config.portal_dir %}/static/vendor/js/tag-it.min.js" type="text/javascript" charset="utf-8"></script>
```

Comment:
just to confirm, it's okay that we're removing this reference to tag-it? I noticed it's still in the 'vendor' directory.

Reply:
I'm pretty sure it's removed ... no?

Reply:
Ah! You are right, sorry. I took the blank screen to mean it was another large diff, but it was in fact deleted.